### PR TITLE
Pin CI overlays for 8.20

### PR DESCRIPTION
**Pull request by proux01**
C.f. https://github.com/coq/coq/issues/18882

### CODE REVIEW

**Comment by ppedrot** [comment]
At least VST and itauto will need a local fix.

**Comment by proux01** [comment]
Yes, vscoq will also need an update

File: dev/ci/ci-basic-overlay.sh [hunk 1]
```diff
@@ -128,319 +128,319 @@ project stdpp "https://gitlab.mpi-sws.org/iris/stdpp" ""
...
 ########################################################################
-project coquelicot "https://gitlab.inria.fr/coquelicot/coquelicot" "master"
+project coquelicot 'https://gitlab.inria.fr/coquelicot/coquelicot' 'ca1a747aa8b7ccbfa67a55ae5c8e5c8df71cc396'
 # Contact @silene on github
 
 ########################################################################
 # CompCert
 ########################################################################
-project compcert "https://github.com/AbsInt/CompCert" "master"
+project compcert 'https://github.com/AbsInt/CompCert' '8f5fd33fb4f9594dacbc11597335ec362344c08d'
```
Comment (silene) [inline dev/ci/ci-basic-overlay.sh:176]: As pointed it out by @SkySkimmer, the VST failure might be caused by CompCert, so
```suggestion
project compcert 'https://github.com/AbsInt/CompCert' '2ca39a2801d333abcfa3d691620d03abde4e7e37'
```

Comment (proux01) [reply dev/ci/ci-basic-overlay.sh]: If it works here, we'll have to do the same on master and ping Andrew.

**Comment by silene** [comment]
@coqbot run full ci

**Review by silene — APPROVED** [review]

**Comment by silene** [comment]
@coqbot merge now

**Comment by silene** [comment]
@coqbot merge now

**Pull request merged**